cv32e40s/tb/uvmt/uvmt_cv32e40s.flist

```diff
@@ -63,10 +63,12 @@ ${TBSRC_HOME}/core/tb_riscv/riscv_gnt_stall.sv
 ${TBSRC_HOME}/core/tb_riscv/riscv_rvalid_stall.sv
 ${TBSRC_HOME}/core/tb_riscv/riscv_random_interrupt_generator.sv
```

Comment:
I think we can remove these "core" verification components from this flist as they are no longer used in the E40S UVM environment.

Reply:
Agreed. If we do "package [the assertion files] separately" in a new PR, then we can maybe remove these "core" components from the uvmt flist at the same time?

Reply:
Agreed.